pyanaconda/packaging/yumpayload.py

```diff
@@ -1165,6 +1165,24 @@ def groupDescription(self, groupid):
 
             return (group.ui_name, group.ui_description)
 
+    def groupId(self, group_name):
+        """Translate group name to group ID.
+
+        :param group_name: Valid identifier for group specification.
+        :returns: Group ID or empty string if nothing is found.
+        :raise NoSuchGroup: If group_name doesn't exists.
+        """
+        groups = self._yumGroups
+        if not groups:
+            return ""
```

Comment:
Ok, but we could at least return `None` and check the id before we modify `self._addonStates`.